### PR TITLE
[ES] Add `B` missing stubs (3)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing B stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish.

32 stubs in total, divided into 4 pull requests of 8 stubs each. Continued in: #10444

### CODE REVIEW

File: wiki/Beatmapping/Hitsound/es.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - adiciones de sonido
+  - adición de hitsound
+  - adiciones de hitsounds
+---
+
+# Hitsound
+
+*Véase también: [Agregar hitsounds personalizados](/wiki/Guides/Using_custom_hitsounds)*
+
+Los **hitsounds** son los sonidos que osu! reproduce en respuesta a la entrada del usuario al interactuar con [objetos](/wiki/Gameplay/Hit_object). Los hitsounds se utilizan normalmente para brindar feedback auditiva al jugador para ayudarlo a juzgar su [precisión](/wiki/Gameplay/Accuracy) en relación con la canción.
```
Comment (agatemosu) [inline wiki/Beatmapping/Hitsound/es.md:23]:
```suggestion
Los **hitsounds** son los sonidos que osu! reproduce en respuesta a la entrada del usuario al interactuar con [objetos](/wiki/Gameplay/Hit_object). Los hitsounds se usan normalmente para brindar retroalimentación auditiva al jugador para ayudarlo a juzgar su [precisión](/wiki/Gameplay/Accuracy) en relación con la canción.
```

File: wiki/Beatmapping/Hitsound/es.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+La reproducción del hitsound consiste en una muestra predeterminada llamada *hitnormal*, y cualquier *adición* de una combinación de muestras whistle, finish o clap. Cada muestra es parte de uno de los tres *conjuntos de muestras* que denotan diferentes estilos: `Normal`, `Soft`, o `Drum`.
+
+Los beatmappers pueden reemplazar cualquier muestra de sonido predeterminada con muestras personalizadas usando la carpeta del [beatmap](/wiki/Beatmap). La [skin](/wiki/Skinning) de un jugador también puede reemplazar los hitsounds predeterminados en todos los beatmaps. Los detalles sobre las muestras de los hitsound se pueden encontrar en el artículo [hitsounds en el skinning](/wiki/Skinning/Sounds#hitsounds).
```
Comment (agatemosu) [inline wiki/Beatmapping/Hitsound/es.md:27]:
```suggestion
Los beatmappers pueden reemplazar cualquier muestra de sonido predeterminada con muestras personalizadas usando la carpeta del [beatmap](/wiki/Beatmap). La [skin](/wiki/Skinning) de un jugador también puede reemplazar los hitsounds predeterminados en todos los beatmaps. Los detalles sobre las muestras de los hitsounds se pueden encontrar en el artículo [hitsounds en el skinning](/wiki/Skinning/Sounds#hitsounds).
```

File: wiki/Beatmapping/Hitsound/es.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Hitsound pasivo
+
+Si el hitsound se encuentra en una parte de un objeto en el que no se puede hacer clic, por ejemplo, un slider inverso o el final de un spinner, se considera *pasivo*. Estos siempre se reproducen al mismo tiempo mientras el jugador interactúa con el objeto, a diferencia de los hitsounds activos. Los hitsounds pasivos no necesitan tener un impacto claro o ser audibles, pero a menudo usan los mismos sonidos que los hitsounds activos cuando aterrizan en tiempos distintos para ayudar al jugador a mantener el ritmo.
+
+## Storyboards con hitsounds
+
+Se pueden agregar muestras de sonido a través del storyboarding para imitar a los hitsounds. Sin embargo, a diferencia de los hitsounds reales, estos siempre se reproducen en el momento dado, independientemente de si el jugador golpeó el objeto. Por ese motivo, muchas veces no se recomiendan. Algunos mappers los utilizan para mantener patrones de hitsounds entre objetos.
```
Comment (agatemosu) [inline wiki/Beatmapping/Hitsound/es.md:41]:
```suggestion
Se pueden agregar muestras de sonido a través del storyboarding para imitar a los hitsounds. Sin embargo, a diferencia de los hitsounds reales, estos siempre se reproducen en el momento dado, independientemente de si el jugador golpeó el objeto. Por ese motivo, muchas veces no se recomiendan. Algunos mappers los usan para mantener patrones de hitsounds entre objetos.
```

File: wiki/Beatmapping/Hitsound/es.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Keysound
+
+Un keysound es una muestra de hitsound que es extremadamente similar a la canción o está tomada directamente de ella y se utiliza para replicar el tono de las notas de la canción. Este método de hitsounding generalmente proporciona poco feedback al jugador, pero puede hacer que jugar ciertos beatmaps sea más interesante y hacer que secciones de un mapa se destaquen cuando se aplica bien.
```
Comment (agatemosu) [inline wiki/Beatmapping/Hitsound/es.md:45]:
```suggestion
Un keysound es una muestra de hitsound que es extremadamente similar a la canción o está tomada directamente de ella y se usa para replicar el tono de las notas de la canción. Este método de hitsounding generalmente proporciona poca retroalimentación al jugador, pero puede hacer que jugar ciertos beatmaps sea más interesante y hacer que secciones de un mapa se destaquen cuando se aplica bien.
```

**Review by agatemosu — COMMENTED** [review]
the last article remaining is tech maps

File: wiki/Beatmap/Technical_maps/es.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+Buenos ejemplos de mapas técnicos que se ajustan a esta definición son [Silentroom - Nhelv (Nyxa) \[iniquitatem\]](https://osu.ppy.sh/beatmapsets/917915#osu/2009432) y [RUMI - Densetsu no Matsuri (Net0) \[Oni\]](https://osu.ppy.sh/beatmapsets/781683#osu/1641637).
+
+### Definición más restrictiva/específica
+
+*Aviso: La definición «más restrictiva/específica» requiere que **todos** los aspectos enumerados estén presentes para poder definir un beatmap como técnico.*
+
+- Formas irregulares de sliders
+- Cambios rápidos y bruscos en la velocidad del slider
+- Sliders SV
+- Patrones difíciles de leer (a menudo descritos como de «flow» difícil) (se muestra un ejemplo a continuación)
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:54]:
```suggestion
- Patrones difíciles de leer (a menudo descritos como de «flow» difícil) (ejemplo mostrado a continuación)
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+## Definiciones
+
+Debido a la naturaleza amplia del término «tech map», existen muchos aspectos diferentes que podrían describirlo. Las listas siguientes describen diferentes definiciones de «tipos» populares de jugadores.
+
+### Definición más amplia
+
+*Aviso: La definición «más amplia» solo requiere que uno o dos de los aspectos enumerados estén presentes para poder definir un beatmap como técnico.*
+
+- Patrones no genéricos o difíciles de leer (se muestra un ejemplo a continuación)
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:32]:
```suggestion
- Patrones no genéricos o difíciles de leer (ejemplo mostrado a continuación)
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  - Un «flow» difícil para el beatmap
+- Sliders SV
+  - Estos pueden contener slider art
+- Formas irregulares de sliders
+- Cambios rápidos y bruscos en la velocidad del slider
+- Alta intensidad general
+  - Extremadamente denso en el recuento de objetos en gran parte del mapa (excluyendo [streams](/wiki/Beatmap/Pattern/osu!/Stream))
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:39]:
```suggestion
  - Extremadamente denso en el número de objetos en gran parte del mapa (excluyendo [streams](/wiki/Beatmap/Pattern/osu!/Stream))
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  - SV
+  - term
+  - técnico
+  - término
+  - deslizador
+---
+
+# Mapas técnicos
+
+*Aviso: Para el propósito de este artículo, el término «tech map» se tratará como un término general que incluye todas estas definiciones.*
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:16]:
```suggestion
*Aviso: Para el propósito de este artículo, el término «mapa técnico» se tratará como un término general que incluye todas estas definiciones.*
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+**Mapa técnico** (a menudo abreviado como «tech maps») es un término que no tiene una definición clara y acordada entre la comunidad de osu!. Dependiendo del usuario, un «tech map» puede ser tan amplio como un [beatmap](/wiki/Beatmap) que simplemente contiene [patrones](/wiki/Beatmap/Pattern) no genéricos, o tan específico como un beatmap que es denso en número de [objetos](/wiki/Gameplay/Hit_object) y contiene muchas formas irregulares de [sliders](/wiki/Gameplay/Hit_object/Slider) que cambian rápida y drásticamente las velocidades (a menudo a alta intensidad).
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:18]:
```suggestion
**Mapa técnico** (a menudo abreviado en inglés como «tech maps») es un término que no tiene una definición clara y acordada entre la comunidad de osu!. Dependiendo del usuario, un «mapa técnico» puede ser tan amplio como un [beatmap](/wiki/Beatmap) que simplemente contiene [patrones](/wiki/Beatmap/Pattern) no genéricos, o tan específico como un beatmap que es denso en número de [objetos](/wiki/Gameplay/Hit_object) y contiene muchas formas irregulares de [sliders](/wiki/Gameplay/Hit_object/Slider) que cambian rápida y drásticamente las velocidades (a menudo a alta intensidad).
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Los mapas técnicos suelen asociarse con géneros de canciones que ofrecen sonidos complejos y de ritmo rápido (por ejemplo, drum-and-bass, dubstep y drumstep), ya que a menudo le permiten al mapper explorar todos y cada uno de los sonidos individuales en detalle para beneficio del beatmap. Aunque no se requiere explícitamente música de este tipo.
+
+Independientemente de la definición, muchos jugadores afirman que los tech maps son injustos (en términos de habilidad) debido al principal aumento de la dificultad que se deriva principalmente o únicamente de patrones confusos o sliders irregulares. Asimismo, esto significa que no son muy populares para ganar grandes cantidades de [pp](/wiki/Performance_points) a la vez debido a la forma en que funciona el sistema actual y cómo infravalora los beatmaps de esta naturaleza.
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:22]:
```suggestion
Independientemente de la definición, muchos jugadores afirman que los mapas técnicos son injustos (en términos de habilidad) debido al principal aumento de la dificultad que se deriva principalmente o únicamente de patrones confusos o sliders irregulares. Asimismo, esto significa que no son muy populares para ganar grandes cantidades de [pp](/wiki/Performance_points) a la vez debido a la forma en que funciona el sistema actual y cómo infravalora los beatmaps de esta naturaleza.
```

File: wiki/Beatmap/Technical_maps/es.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+## Definiciones
+
+Debido a la naturaleza amplia del término «tech map», existen muchos aspectos diferentes que podrían describirlo. Las listas siguientes describen diferentes definiciones de «tipos» populares de jugadores.
```
Comment (agatemosu) [inline wiki/Beatmap/Technical_maps/es.md:26]:
```suggestion
Debido a la naturaleza amplia del término «mapa técnico», existen muchos aspectos diferentes que podrían describirlo. Las listas siguientes describen diferentes definiciones de «tipos» populares de jugadores.
```

**Comment by agatemosu** [comment]
and I say this for the third time: this commit https://github.com/ppy/osu-wiki/pull/10443/commits/6b319e28d660dff76a5c108c81f5e189bd49e67f can be dropped?

**Comment by TicClick** [comment]
^ while I can do that, I see no point in pruning a single commit while we have dozens, no, hundreds of them in `master`

**Review by TicClick — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**